source/js/donate-modal/donate-modal.jsx

```diff
@@ -2,58 +2,47 @@ import React from "react";
 import ReactDOM from "react-dom";
 import ReactGA from "../react-ga-proxy.js";
 
-const KEY_STATE = `donate modal state`;
-const KEY_TIMER = `donate modal timer`;
-const DELAY = 10000; // in ms
-const TIMER_INCREMENT = 1000; // in ms
+const DISMISSED_CHECK_KEY = "fundraising-banner";
+const MILLISECONDS = 1000;
+const DAY = 24 * 3600 * MILLISECONDS;
+const localStorage = typeof window === "undefined" ? {} : window.localStorage;
 
 class DonateModal extends React.Component {
   constructor(props) {
     super(props);
 
-    let dismissed = !!sessionStorage.getItem(KEY_STATE) || false;
-    let prevTimer = this.getPrevTimer();
+    const INITIAL_DELAY = parseInt(this.props.delay) || 10 * MILLISECONDS; // 10 seconds, in ms
+    const DISMISSAL_DELAY = parseInt(this.props.hideFor) || 7; // one week, in days
```

Comment:
I was thinking maybe we could rename these two variables into `INITIAL_DELAY_IN_MILLISECONDS` and `DISMISSAL_DELAY_IN_DAYS` for clarity?

Reply:
good call